bin/resources/GameIndex.yaml

```diff
@@ -76,6 +79,9 @@ PAPX-90516:
     mipmap: 1
     textureInsideRT: 1
     autoFlush: 1
+PAPX-90517:
+  name: "Prince of Persia: Jikan no Suna [Trial]"
```

Comment:
Hyphen instead of colon please

Reply:
gsHWFixes:
    autoFlush: 1
    
If you can read romaji it says Sands of time and this is what the other serials contain.